DomainContent/LoadTest/Trivia/trivia.js

```diff
@@ -0,0 +1,384 @@
+            };
+    
+            return (!value) ? value : String(value).replace(charToEntityRegex, htmlEncodeReplaceFn);
+        }
+    
+        function htmlDecode(value) {
+            var htmlDecodeReplaceFn = function(match, capture) {
+                return (capture in entityToChar) ? entityToChar[capture] : String.fromCharCode(parseInt(capture.substr(2), 10));
+            };
+    
+            return (!value) ? value : String(value).replace(entityToCharRegex, htmlDecodeReplaceFn);
+        }
+    
+        resetCharacterEntities();
+    
+        return {
+            htmlEncode: htmlEncode,
+            htmlDecode: htmlDecode
+        };
+    })();
+
+    function begin() {
+        playSound(GAME_BEGIN_SOUND);
+        findTargets();
+        clearGame();
+    }
+  
+    function findTargets() {
+        Entities.findEntities(MyAvatar.position, SEARCH_RADIUS).forEach(function(element) {
+            var name = Entities.getEntityProperties(element, ['name']).name;
+            switch (name) {
+                case "Trivia Question":
+                    questionText = element;
+                    break;
+                case "Trivia Answer":
+                    answerText = element;
+                    break;
+                case "Trivia Choice Text 1":
+                    choiceTexts[0] = element;
+                    break;
+                case "Trivia Choice Text 2":
+                    choiceTexts[1] = element;
+                    break;
+                case "Trivia Choice Text 3":
+                    choiceTexts[2] = element;
+                    break;
+                case "Trivia Choice Text 4":
+                    choiceTexts[3] = element;
+                    break;
+                case "Trivia Podium Question":
+                    podiumQuestions.push(element);
+                    break;
+                case "Trivia Podium Reveal":
+                    podiumReveals.push(element);
+                    break;
+            }
+        });
+    }
+
+    function onClicked() { 
+        if (open) {
+            tablet.gotoHomeScreen();
+        } else {
+            tablet.gotoWebScreen(appPage);
+        }
+    }
+
+    function getQuestion() {
+        try {
+            var triviaURL = "https://opentdb.com/api.php?amount=1";
+            if (type) {
+                triviaURL = triviaURL + "&type=" + type;
+            }
+            if (difficulty) {
+                triviaURL = triviaURL + "&difficulty=" + difficulty;
+            }
+            if (category) {
+                triviaURL = triviaURL + "&category=" + category;
+            }
+            request(triviaURL, function (error, data) {
+                if (!error) {
+                    tablet.emitScriptEvent(JSON.stringify(data.results[0]));
+                    triviaData = data.results;
+                }
+            });
+        } catch (err) {
+            print("Could not get domain data using userData domainAPIURL");
```

Comment:
You might want to consider emitting this as well to the tablet to display an error visibly to the trivia master if there is an issue with the API call.